docs/source/package_reference/logging_methods.rst

```diff
@@ -1,8 +1,38 @@
 Logging methods
 ----------------------------------------------------
 
-`datasets` tries to be very transparent and explicit about it's inner working bt this can be quite verbose at some time.
-A series of logging methods let you easily adjust the level of logging of the whole library.
+`datasets` tries to be very transparent and explicit about its inner working, but this can be quite verbose at times.
+
+A series of logging methods let you easily adjust the level of verbosity of the whole library.
+
+Currently the default verbosity of the library is ``INFO``.
+
+To change the level of verbosity, just use one of the direct setters. For instance, here is how to change the verbosity to the INFO level.
+
+.. code-block:: python
+
+    import datasets
+    datasets.logging.set_verbosity_info()
+
+You can also use the environment variable ``DATASETS_VERBOSITY`` to override the default verbosity. You can set it to one of the following: ``debug``, ``info``, ``warning``, ``error``, ``critical``. For example:
+
+.. code-block:: bash
+               
+    DATASETS_VERBOSITY=error ./myprogram.py
+
+All the methods of this logging module are documented below, the main ones are
+:func:`datasets.logging.get_verbosity` to get the current level of verbosity in the logger and
+:func:`datasets.logging.set_verbosity` to set the verbosity to the level of your choice. In order (from the least
+verbose to the most verbose), those levels (with their corresponding int values in parenthesis) are:
+
+- :obj:`datasets.logging.CRITICAL` or :obj:`datasets.logging.FATAL` (int value, 50): only report the most
+  critical errors.
+- :obj:`datasets.logging.ERROR` (int value, 40): only report errors.
+- :obj:`datasets.logging.WARNING` or :obj:`datasets.logging.WARN` (int value, 30): only reports error and
+  warnings. This the default level used by the library.
```

Comment:
but line 8 mentions that the default level is `INFO` 🤔

Reply:
Right !
Let's change to INFO for now as well in the docs @stas00

Reply:
Great catch, @stefan-it . I removed that note, since it's already written above it. So having one place is less error-prone.